odoo/addons/base/models/ir_actions.py

```diff
@@ -272,6 +272,9 @@ def _check_view_mode(self):
     target = fields.Selection([('current', 'Current Window'), ('new', 'New Window'), ('inline', 'Inline Edit'), ('fullscreen', 'Full Screen'), ('main', 'Main action of Current Window')], default="current", string='Target Window')
     view_mode = fields.Char(required=True, default='tree,form',
                             help="Comma-separated list of allowed view modes, such as 'form', 'tree', 'calendar', etc. (Default: tree,form)")
+    default_mobile_mode = fields.Char(string="Default Mobile View", default="kanban", help="View mode override in mobile and small "
```

Comment:
I don't think the prefix `default_` brings anything useful here. Also, it's a bit weird that `view_mode` is a comma separated list of view types and `default_mobile_mode` is a single view type. This model didn't age well... `view_mode` (why "mode" btw?), `view_ids`, `views`, `view_id`... Also `usage`, is it even used?